pkg/controller/directvolumemigration/namespace.go

```diff
@@ -22,13 +23,14 @@ func (t *Task) ensureDestinationNamespaces() error {
 	}
 
 	// Get list namespaces to iterate over
-	nsMap := t.getPVCNamespaceMap()
-	for ns, _ := range nsMap {
+	nsMap := t.getBothPVCNamespaceMap()
+	for bothNs, _ := range nsMap {
+		nsNames := strings.Split(bothNs, ":")
```

Comment:
If I understand this correctly, `bothNs` is a key in the map which is stored after splittting the string with `:` delimeter. Looks like this operation will be redundant. For clarity, I suggest iterating over the map as:

```golang
for originalNs, mappedNs := range nsMap {
```

Reply:
Sorry, I got confused between the two different methods. The above isn't true. Please ignore.

Reply:
Yeah, unfortunately golang doesn't allow slices as array keys. I want to index this by (srcNS, destNS) pairs, but the only way to do that is embedding it in a string.